src/main/java/frc/robot/Robot.java

```diff
@@ -35,6 +43,12 @@ public void robotInit() {
         // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
         // autonomous chooser on the dashboard.
         m_robotContainer = new RobotContainer();
+
+        mCompressor = new Compressor(0);
```

Comment:
```suggestion
        mCompressor = new Compressor();
```
The default port is zero so you can get away with this.